### PR TITLE
QtPBFImagePlugin: new recipe

### PR DESCRIPTION
**Pull request by sikmir**

### CODE REVIEW

File: dev-qt/qtpbfimageplugin/qtpbfimageplugin-1.2.recipe [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	qtpbfimageplugin$secondaryArchSuffix = $portVersion
+	lib:libpbf$secondaryArchSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libprotobuf_lite$secondaryArchSuffix
```
Comment (diversys) [inline dev-qt/qtpbfimageplugin/qtpbfimageplugin-1.2.recipe]: Please sort alphabetically.

File: dev-qt/qtpbfimageplugin/qtpbfimageplugin-1.2.recipe [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+	lib:libGL$secondaryArchSuffix
+	lib:libQt5Core$secondaryArchSuffix
+	lib:libQt5Gui$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libprotobuf_lite$secondaryArchSuffix
```
Comment (diversys) [inline dev-qt/qtpbfimageplugin/qtpbfimageplugin-1.2.recipe]: Ditto.

File: dev-qt/qtpbfimageplugin/qtpbfimageplugin-1.2.recipe (hunk 1 above)
Comment (sikmir) [reply dev-qt/qtpbfimageplugin/qtpbfimageplugin-1.2.recipe]: Fixed.

File: dev-qt/qtpbfimageplugin/qtpbfimageplugin-1.2.recipe (hunk 2 above)
Comment (sikmir) [reply dev-qt/qtpbfimageplugin/qtpbfimageplugin-1.2.recipe]: Fixed.

**Pull request merged**